dashboard/src/components/Header/Header.tsx

```diff
@@ -139,7 +139,9 @@ class Header extends React.Component<IHeaderProps, IHeaderState> {
                         <NavLink to={reposPath}>App Repositories</NavLink>
                       </li>
                       <li role="none">
-                        <NavLink to="/config/brokers">Service Brokers</NavLink>
+                        <NavLink to={app.config.brokers(clusters.currentCluster)}>
+                          Service Brokers
+                        </NavLink>
```

Comment:
Should this link be displayed only if the currentCluster is the default one or is it actually expected that users reach this endpoint in order to get the error message and understand why is not available?

Reply:
Yes, it is expected that the menu item is there (when service brokers are configured), rather than not displaying options for reasons which are not clear to the user (seems to be Kubeapps UX strategy - in form the user of what RBAC is missing).

That said, I did land an earlier PR which does not display this menu option unless service brokers are configured.